samples/04.0-file-transfer/system-tests/kubernetes-test.sh

```diff
@@ -0,0 +1,35 @@
+#/bin/sh
```

Comment:
Have you considered writing an integration test with Java instead of a bash script?